py-polars/polars/datatypes/constructor.py

```diff
@@ -111,7 +111,7 @@ def numpy_values_and_dtype(
         else:
             raise ValueError(
                 "'D' (datetime only), 'ms', 'us', and 'ns' resolutions are supported when converting from numpy.{datetime64,timedelta64}"
-                "\n\nPlease cast to the closest supported unit before converting"
+                "\n\nPlease cast to the closest supported unit before converting."
```

Comment:
Splitting the message in this way doesn't make sense. Clearly this whole thing is the 'long explanation', it's missing a proper short summary (e.g. `converting from NumPy failed` or whatever is going on here).